tests/playwright/Report.md

```diff
@@ -0,0 +1,124 @@
+Extendable
+Adding helper functions, or more complicated sets and combinations of existing commands is simple and really useful
+
+Compatible
+WebdriverIO can be run on the WebDriver Protocol for true cross-browser testing as well as Chrome DevTools Protocol for Chromium-based automation using Puppeteer.
+
+Feature Rich
+The huge variety of community plugins allows you to easily integrate and extend your setup to fulfill your requirements.
+
+The above points are just a few of WebdriverIO capability. To explore more WebdriverIO capability, you can check out the [WebdriverIO](https://webdriver.io/) documentation and the [Wikipedia Repo](https://github.com/wikimedia/mediawiki/tree/master/tests/selenium).
+
+[Playwright](https://github.com/microsoft/playwright) is a Node.js library to automate Chromium, Firefox, and WebKit with a single API. Playwright is known for the following capability;
+
+Test across all modern browsers
+Single API to automate Chromium, Firefox, and WebKit.
+
+Use in your preferred language
+Use the Playwright API in JavaScript & TypeScript, Python, C# and, Java.
+
+Automate without trade-offs
+Capable automation for single-page apps that rely on the modern web platform.
+
+The above points are just a few capabilities. To explore more on Playwright capability, you can check out the [Playwright](https://playwright.dev/) documentation and this [ forked Wikipedia Repo](https://github.com/hariclerry/mediawiki/tree/master/tests/playwright).
+
+While working with Playwright for the past two months, it has come across as easy to install and setup. The package takes care of installing all the browsers(Chromium, Firefox, and WebKit). Capturing Screenshots is an out-of-the-box experience. However, Video recording requires separate installation of [playwright-video and ffmpeg](https://playwright.tech/blog/record-your-browser-tests-with-playwright) but they all blend in with Playwright seamlessly.
+
+Below are some of the benefits I have experienced and seen with using playwright;
+
+- Console debug option which is very useful in an event when tests are failing anonymously.
+- Scenarios that span multiple pages, domains, and iframes
+- Auto-wait for elements to be ready before executing actions (like click, fill)
+- Intercept network activity for stubbing and mocking network requests
+- Seamless integration with Jest.
+
+Sample Page screenshot code
+
+This code snippet navigates to wikipedia.org in WebKit and saves the screenshot.
+
+    `const playwright = require('playwright');
+
+    (async () => {
+        const browser = await webkit.launch();
+        const context = await browser.newContext();
+        const page = await context.newPage();
+        await page.goto('https://www.wikipedia.org/');
+        await page.screenshot({ path: 'wikipedia-home-page.png' });
+        await browser.close();
+    })();`
```

Comment:
Something went wrong with code formatting. I don't think you need both 4 space indentation and backticks. I would recommend fenced code blocks:

https://github.github.com/gfm/#fenced-code-blocks